interface-definitions/container.xml.in

```diff
@@ -368,9 +368,33 @@
           </leafNode>
         </children>
       </tagNode>
-      <leafNode name="registry">
+      <tagNode name="registry">
```

Comment:
The default value for the registry should be `docker.io quay.io` if not configured explicitly.
But if the `registry` is configured, It shouldn't use the default values but only configured registry entries.

Reply:
I see, where's the place that I should store `docker.io quay.io`, is it inside .py file or j2 file?

Reply:
@c-po as I understand it possible to use default values with tagNode? Or  is it better to use Python for this case?

Reply:
We must always use `<defaultValue>` to have the defaults in one unique place. I think it is currently not possible and needs a schema extension.

Reply:
Will be added in https://vyos.dev/T5079